DataFormats/CTPPSReco/src/CTPPSPixelLocalTrack.cc

```diff
@@ -0,0 +1,99 @@
+/*
+ *
+* This is a part of CTPPS offline software.
+* Author:
+*   Fabrizio Ferro (ferro@ge.infn.it)
+*   Enrico Robutti (robutti@ge.infn.it)
+*   Fabio Ravera   (fabio.ravera@cern.ch)
+*
+*/
+
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrack.h"
+// #include "Rtypes.h"
+
+//----------------------------------------------------------------------------------------------------
+
+math::Matrix<2,2>::type CTPPSPixelLocalTrack::trackPointInterpolationCovariance(float z) const
```

Comment:
The same type is also known as AlgebraicSymMatrix22
It may be easier to read and type in the code